src/display.cpp

```diff
@@ -454,6 +454,16 @@ void setToFullWindow()
   display.setFullWindow();
 }
 
+void setToPartialWindow(int16_t x, int16_t y, int16_t w, int16_t h)
```

Comment:
Just a minor, but please follow the args style:
```suggestion
void setToPartialWindow(int16_t xCord, int16_t yCord, int16_t width, int16_t height)
```